### PR TITLE
fix: construct urls using URL class

### PR DESCRIPTION
**Pull request by JivusAyrus**
## TODO

- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

File: controlplane/src/core/bufservices/PlatformService.ts [hunk 1]
```diff
@@ -9921,8 +9921,14 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
           name: provider.name,
           endpoint: provider.endpoint,
           loginURL: `${opts.webBaseUrl}/login?sso=${provider.alias}`,
-          signInRedirectURL: `${opts.keycloakApiUrl}/realms/${opts.keycloakRealm}/broker/${provider.alias}/endpoint`,
-          signOutRedirectURL: `${opts.keycloakApiUrl}/realms/${opts.keycloakRealm}/broker/${provider.alias}/endpoint/logout_response`,
+          signInRedirectURL: new URL(
```
Comment (StarpTech) [inline controlplane/src/core/bufservices/PlatformService.ts:9924]: The proto defines a string. Tsc correctly infers that it is stringified?

Comment (JivusAyrus) [reply controlplane/src/core/bufservices/PlatformService.ts:9924]: You mean does tsc understand that its a string?

Comment (JivusAyrus) [reply controlplane/src/core/bufservices/PlatformService.ts:9923]: Yep, i do toString()

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**